app/helpers/admin/document_collection_group_memberships_helper.rb

```diff
@@ -1,14 +1,17 @@
 module Admin::DocumentCollectionGroupMembershipsHelper
+  UNAVAILABLE_DOCUMENT_TITLE = "Unavailable Document".freeze
+
   def document_collection_group_member_title(membership)
     return membership.non_whitehall_link.title if membership.non_whitehall_link
+    return UNAVAILABLE_DOCUMENT_TITLE unless membership.document&.latest_edition
```

Comment:
Does the "Unavailable Document" label need to retain its **bold** styling? Or has that requirement been dropped?

```suggestion
    return tag.span(UNAVAILABLE_DOCUMENT_TITLE, class: "govuk-!-font-weight-bold") unless membership.document&.latest_edition
```

Reply:
There wasn't!! I completely missed - thank you! will change before merge